manifests/blacklist.pp

```diff
@@ -1,10 +1,13 @@
 # ex: si ts=4 sw=4 et
 
 define shorewall::blacklist (
-    $type         = 'ipv4',
+    $action        = 'DROP',
+    $type          = 'ipv4',
+    $source        = 'all',
     $address       = '',
+    $dest          = '$FW',
```

Comment:
It looks like `$source`, `$dest`, and `$action` need to cause a catalog compilation error when `$::shorewall_version < 40425` since these values are unused in the older `blacklist` configuration file.